src/jit/codegen.h

```diff
@@ -750,7 +750,12 @@ protected :
             struct
             {
                 regNumberSmall  scRegNum;
-                regNumberSmall  scOtherReg; // used for "other half" of long var
+                
+                // Used for:
+                //  - "other half" of long var on OSs with 32 bit size registers - x86.
+                //  - for System V structs it stores the second register 
+                //    used to pass a register passed struct.
```

Comment:
We won't have a problem as far as I recall. Structs for System V on x86 do not use multi-register passing.